client/globals.d.ts

```diff
@@ -8,6 +8,7 @@ declare const wcpaySettings: {
 	connectUrl: string;
 	isSubscriptionsActive: boolean;
 	featureFlags: {
+		customDepositSchedules?: boolean;
```

Comment:
Looks like this line was inadvertently added while merging `develop` after #5008.

Reply:
Yep this was removed as part of this commit: https://github.com/Automattic/woocommerce-payments/pull/5891/commits/3d7768e6ca71ae9492a885d627866e715d0e9453 when i updated to the latest `develop` after #5008